app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/TicketDao.kt

```diff
@@ -13,6 +10,8 @@ interface TicketDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insert(vararg ticket: Ticket)
 
+    @SuppressWarnings(RoomWarnings.CURSOR_MISMATCH)
+    @Transaction
```

Comment:
I assume, Room changed its behavior and complaints about this query now? If yes this could use a //todo

Reply:
Yes, Room is complaining. I will add a TODO.